### PR TITLE
fix(lib): add request url to response object and check response conte…

### PR DESCRIPTION
**Pull request by ashihminaval**
…nt type

### CODE REVIEW

File: src/native-http-backend.ts [hunk 1]
```diff
@@ -121,6 +132,7 @@ export class NativeHttpBackend implements HttpBackend {
                             body,
                             headers: new HttpHeaders(response.headers),
                             status: response.status,
+                            url: req.url,
```
Comment (sneas) [inline src/native-http-backend.ts]: Should it be just `url`?

File: src/native-http-backend.ts [hunk 2]
```diff
@@ -134,6 +146,7 @@ export class NativeHttpBackend implements HttpBackend {
                             error: body,
                             headers: new HttpHeaders(response.headers),
                             status: response.status,
+                            url: req.url,
```
Comment (sneas) [inline src/native-http-backend.ts]: Looks like it should be `url` instead of `req.url`

**Comment by sneas** [comment]
Thank you for the PR! I added some comments.

**Comment by sneas** [comment]
Well done @ashihminaval ! 👍

**Comment by ashihminaval** [comment]
> Well done @ashihminaval ! 👍

Thank you!

**Pull request merged**